playbooks/kubevirt-web-ui/config.yml

```diff
@@ -1,14 +1,19 @@
 ---
-#- import_playbook: ../init/main.yml
-#  vars:
-#    l_init_fact_hosts: "oo_masters_to_config"
-#    l_openshift_version_set_hosts: "oo_masters_to_config:!oo_first_master"
-#    l_sanity_check_hosts: "{{ groups['oo_masters_to_config'] }}"
+- name: Kubevirt Web UI
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  tasks:
+  - import_role:
+      name: kubevirt_web_ui
```

Comment:
Looks like this should get moved up to ```playbooks/kubevirt-web-ui.yml``` so CI looks in the right place for the role.